interface/usergroup/addrbook_edit.php

```diff
@@ -29,6 +29,7 @@
  $info_msg = "";
 
  function invalue($name) {
+  if (!$_POST[$name]) return "''";
   $fld = add_escape_custom(trim($_POST[$name]));
   return "'$fld'";
```

Comment:
@ophthal
Is the suffix stuff needed for the eye form? (I am not seeing this in the database)